mobile/docs/root/api/starting_envoy.rst

```diff
@@ -363,6 +363,36 @@ Specify whether to enable transparent response Brotli decompression. Defaults to
 Default values from the `brotli decompressor proto <https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/compression/brotli/decompressor/v3/brotli.proto>`_
 are used.
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``enableHttp3``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Specify whether to enable HTTP/3. Defaults to false. Only available when the Envoy Mobile build has HTTP/3 included.
+
+**Example**::
+
+  // Kotlin
+  builder.enableHttp3(true)
```

Comment:
Do we have this function in Kotlin? I don't see it in library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

Reply:
Surprised we don't have it in Kotlin. Added the API.